frontend/src/pages/home.jsx

```diff
@@ -0,0 +1,162 @@
+                    </div>
+                    <div className="grid-x grid-margin-x">
+                        <div className="cell large-5 medium-6 large-offset-1">
+                            <p>
+                                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                                do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+                                Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+                                laboris nisi ut aliquip ex ea commodo consequat. Duis aute 
+                                irure dolor in reprehenderit in voluptate velit esse cillum dolore
+                                eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non 
+                                , sunt in culpa qui officia deserunt mollit anim id est laborum." 
+                            </p>
+                            <br/>
+                            <p>
+                                Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore 
+                                eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt 
+                                in culpa qui officia deserunt mollit anim id est laborum.
+                            </p>
+                        </div>
+                        <div className="cell large-5 medium-6">
+                            <p>
+                                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt 
+                                ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+                                laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit 
+                                in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat c
+                                upidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                            </p>
+                            <br/>
+                        </div> 
+                        <div className="cell large-offset-4" style={{position:"absolute", display: "flex", marginTop:"150px" }}>
+                            <img src={Triangle} alt=""/>
+                        </div>
+                    </div>   
+                </div>     
+            </div>   
+            <div style={{background:"#FFEBF3"}}>
+                <div className="feature">
+                    <div className="cell small-7">
+                        <h2>
+                            Featured Stories
+                        </h2>
+                        <h6 style={{color:"#6E6D7A"}}>
+                            Read the impact stories from around the world
+                        </h6>
+                        <br/>
+                        <ul className="list__item-container large-offset-1">
```

Comment:
This markup will create a problem while making the site responsive. 

It should be like this
```
<div className="grid-container">
    <div className="grid-x grid-margin-x">
        <div className="cell large-12">
            //heading and text will come here
        </div>

         <div className="cell large-4">
            <BlogCard />
        </div>
        <div className="cell large-4">
            <BlogCard />
        </div> 
        <div className="cell large-4">
            <BlogCard />
        </div>
    </div>
</div>

Reply:
Got it, thank you!

Reply:
Okat, understood 👍